config/initializers/training_routes.rb

```diff
@@ -47,11 +47,11 @@
 }.freeze
 
 TRAINING_ROUTES_FOR_TRAINEE = TRAINING_ROUTES.select { |training_route|
-  TRAINING_ROUTE_ENUMS.values_at(:assessment_only, :provider_led_postgrad, :early_years_undergrad, :school_direct_tuition_fee, :school_direct_salaried, :early_years_assessment_only, :early_years_salaried, :early_years_postgrad, :pg_teaching_apprenticeship).include? training_route
+  TRAINING_ROUTE_ENUMS.values_at(:assessment_only, :provider_led_postgrad, :early_years_undergrad, :school_direct_tuition_fee, :school_direct_salaried, :early_years_assessment_only, :early_years_salaried, :early_years_postgrad, :pg_teaching_apprenticeship, :hpitt_postgrad).include? training_route
 }.freeze
 
 TRAINING_ROUTES_FOR_COURSE = TRAINING_ROUTES.select { |training_route|
-  TRAINING_ROUTE_ENUMS.values_at(:provider_led_postgrad, :school_direct_tuition_fee, :school_direct_salaried, :pg_teaching_apprenticeship).include? training_route
+  TRAINING_ROUTE_ENUMS.values_at(:provider_led_postgrad, :school_direct_tuition_fee, :school_direct_salaried, :pg_teaching_apprenticeship, :hpitt_postgrad).include? training_route
```

Comment:
```suggestion
  TRAINING_ROUTE_ENUMS.values_at(:provider_led_postgrad, :school_direct_tuition_fee, :school_direct_salaried, :pg_teaching_apprenticeship).include? training_route
```

Don't think `:hpitt_postgrad` is applicable

Reply:
Not having this as a route will break on https://github.com/DFE-Digital/register-trainee-teachers/blob/7d7da41540282ec51eed952c6aee5e82c908c468/app/lib/hpitt.rb#L51
We might need that one to be amended as well, but unsure of what's the right approach.

Reply:
That's pointless assumption...

https://github.com/DFE-Digital/register-trainee-teachers/blob/7d7da41540282ec51eed952c6aee5e82c908c468/app/services/teacher_training_api/import_course.rb#L68-L78

Reply:
Updated.